CMakeLists.txt

```diff
@@ -7,6 +7,8 @@ project(CoreCLR)
 # Enable @rpath support for shared libraries.
 set(MACOSX_RPATH ON)
 
+#SET_PROPERTY(GLOBAL PROPERTY GLOBAL_DEPENDS_DEBUG_MODE 1)
```

Comment:
Could you please remove this commented out line? I have forgotten to do so, it was just for debugging purposes.